features/standard/ContentTypeFields.feature

```diff
@@ -132,10 +132,10 @@ Feature: Content fields setting and editing
       | label    | <label1> | fieldType   |
       | Field    | <value1> | <fieldName> |
     Examples:
-      | fieldInternalName | fieldName     | fieldSettings                                               | label1 | value1                 |
-      | ezcontentquery    | Content query | QueryType-EzPlatformAdminUi:MediaSubtree,ContentType-folder | value  | Media,Files,Multimedia |
+      | fieldInternalName | fieldName     | fieldSettings                                                                                                  | label1 | value1                  |
+      | ezcontentquery    | Content query | QueryType-Folders under media,ContentType-folder,ItemsPerPage-100,Parameters-contentTypeId:folder;locationId:43| value  | Images,Files,Multimedia |
```

Comment:
Nitpick: `contentTypeId` is a numeric value (1,2,3 etc). Here it should be `contentTypeIdentifier`.